packages/vue/src/components/organisms/SfStoreLocator/SfStoreLocator.vue

```diff
@@ -200,6 +200,7 @@ export default {
     }
   },
   mounted() {
+    import("leaflet/dist/leaflet.css");
```

Comment:
@aniamusial @patzick it's some special reason to import css on mounted? Can we move it  to line 65?

Reply:
For me doesn't matter, copied from Ania PR :) we need to fix the issue with building so from my perspective better to load this CSS too late for a moment than not to fix the build breaking bug.

Reply:
I believe line 65 is a fair point; I didn't want to interfere with the solution from other PR. Ania - is there a reason why not to do this? :)